Code/UnitTests/CoreTest/World/WorldTest.cpp

```diff
@@ -710,6 +766,55 @@ EZ_CREATE_SIMPLE_TEST(World, World)
     }
   }
 
+
+  EZ_TEST_BLOCK(ezTestBlock::Enabled, "Set/GetWorldSimulationEnabled behavior")
+  {
+    ezWorldDesc worldDesc("Test");
+    ezWorld world(worldDesc);
+    EZ_LOCK(world.GetWriteMarker());
+
+    world.SetWorldSimulationEnabled(true);
+
+    auto pModule = world.GetOrCreateModule<SimulationTestModule>();
+
+    // World simulation is active, both calls should update
+    world.Update();
+    EZ_TEST_INT(pModule->m_nTimesAlwaysUpdateCalled, 1);
+    EZ_TEST_INT(pModule->m_nTimesSimulationUpdateCalled, 1);
+
+    // This is a sanity check to ensure the order of the Update() calls is AlwaysUpdate() and then SimulationUpdate()
```

Comment:
If there is no priority given update function in the same phase are sorted by name, so AlwaysUpdate will come before SimulationUpdate. If you want to enforce that use a priority on the desc when registering the update function. Higher priorities (higher number) are called earlier.

Reply:
Fixed in 63e08d6